### PR TITLE
Feature/remove unused code

### PR DESCRIPTION
**Pull request by McDaddy**
## What this PR does / why we need it:
there are so many unless code which would take up unnecessary maintenance, then write a script to find out all of them

1. create ts script to find out the unused files
2. clear unused js/scss/jsx/ts/tsx
3. clear related i18n

## Does this PR introduce a user interface change?
- [ ] Yes(screenshot is required)
- [x] No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/tools/detect-unused-files.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  'gateway-ingress': resolve('./app/modules/msp/monitor/gateway-ingress'),
+  'docker-container': resolve('./app/modules/msp/monitor/docker-container'),
+  'mobile-insight': resolve('./app/modules/msp/monitor/mobile-insight'),
+  'api-insight': resolve('./app/modules/msp/monitor/api-insight'),
+  'trace-insight': resolve('./app/modules/msp/monitor/trace-insight'),
+  'monitor-common': resolve('./app/modules/msp/monitor/monitor-common'),
+  topology: resolve('./app/modules/msp/monitor/topology'),
+  'status-insight': resolve('./app/modules/msp/monitor/status-insight'),
+  'error-insight': resolve('./app/modules/msp/monitor/error-insight'),
+  'monitor-alarm': resolve('./app/modules/msp/monitor/monitor-alarm'),
```
Comment (daskyrk) [inline shell/tools/detect-unused-files.ts:64]: It's better to define the map in webpack.config.js and import to here to keep it sync in two places.

File: shell/tools/traverse-module.ts [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+const CSS_EXTS = ['.css', '.scss'];
+const JSON_EXTS = ['.json'];
+
+let requirePathResolver: (curDir: string, requirePath: string) => string;
+
+const MODULE_TYPES = {
+  JS: 1 << 0,
+  CSS: 1 << 1,
+  JSON: 1 << 2,
+};
```
Comment (daskyrk) [inline shell/tools/traverse-module.ts:32]: Why use this syntax?

Comment (McDaddy) [reply shell/tools/traverse-module.ts:31]: removed this syntax

**Pull request merged**